app/store/keys/keys.reducer.ts

```diff
@@ -16,16 +18,22 @@ export interface KeysState {
   salt?: string;
   decryptionError?: string;
   encryptedMnemonic?: string;
-  stxAddress?: string;
   publicKey?: string;
+  initialized: boolean;
```

Comment:
Maybe we can add a comment somewhere to explain what it means to be 'initialized'.
From what I see it means the password is set and the publicKey is successfully derived.

Reply:
+1 on this naming, not entirely clear what it means

Reply:
what's the difference between this initialized prop and inferring whether a wallet is "initalised" by presence of either `stxAddress` or `publicKey`?

Reply:
I added `initalised` to differentiate between logged-in users (terms accepted, phrase and password entered) and users for who we saved their public key. As we have logged-in users who we still need to ask for their passwords to calculate and persist the public key, does that make sense?

Reply:
In other words, if this flag is true, we show the home page, unless the public key is missing (then we show unlock screen). We can't derive this from public key existence as it'll be missing in both scenarios. Also can't derive it from stxAddress as we no longer persist and it's now derived from public address.